conf.py

```diff
@@ -2,10 +2,13 @@
 
 from __future__ import unicode_literals
```

Comment:
From a [recent discussion at SymPy](https://github.com/sympy/sympy/pull/9692#issuecomment-126162173) I learned that we shouldn't use `unicode_literals` and hence this line should be removed.